source/beamline/BioXAS/BioXASSideBeamline.cpp

```diff
@@ -316,7 +316,7 @@ void BioXASSideBeamline::onValveError()
 
 			current = qobject_cast<CLSBiStateControl *>(valveSet_->at(i));
 
-			if (current->state() == 0) // Closed is 0.
+			if (current && current->state() == 0) // Closed is 0.
```

Comment:
Can this be changed to the more descriptive ```isClosed()```?  In general, we should always use the more descriptive getters rather than comparing against numbers.